schema/src/main/java/org/hl7/v3/ValueAdapter.java

```diff
@@ -22,7 +22,7 @@ public class ValueAdapter extends XmlAdapter<Object, Object> {
 
     @Override
     public Object unmarshal(Object v) {
-        if (v instanceof PQ || v instanceof IVLPQ || v instanceof CV) {
+        if (v instanceof PQ || v instanceof IVLPQ || v instanceof CD) {
```

Comment:
This looks fair but are we sure we will never get a CV instance in the unmarshal?

Reply:
`instanceof CD` covers CV, as CV is a subclass.